File: Codes/phase1/Q1/wrong.py
import math


def calc(c, x, sum):
    while(c > 0):
        sum += c % x
        c = c // x
        sum = calc(c, x, sum)
        return sum
    return sum


c, x = map(int, input().split())
sum = 0
sum = c % x
c = c // x
sum = calc(c, x, sum)
print(sum)

File: Codes/phase1/Q1/wrong.java
package temp;
import java.util.*;
public class Weed {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int c = sc.nextInt();
		int x = sc.nextInt();
		int sum = 0;
	    sum = c % x;
	    c = c / x;
	    sum = calc(c, x, sum);
	    System.out.println(sum);
		
	}
	static int calc(int c, int x, int sum)
	{
	    while(c > 0)
	    {
	        sum += c % x;
	        c = c / x;
	        sum = calc(c, x, sum);
	        return sum;
	    }
	    return sum;
	}
}
